web_programming/fetch_amazon_product_data.py

```diff
@@ -15,17 +15,21 @@
 from bs4 import BeautifulSoup as bs
 
 
-def get_product_info(product="laptop") -> None:
-    # function that will take the product as input and return the product details as output
-    # in the form of a csv file,if no input is given,it will fetch the details of laptop by default
+def get_product_info(product:str = "laptop") -> None:
+    # function that will take the product as input and return the 
+    #product details as output
+    # in the form of a csv file,if no input is given,it 
+    #will fetch the details of laptop by default
```

Comment:
```suggestion
    """
    Take the product as input and return the product details as output in the form
    of a csv file.  If no input is given, it will fetch the details of `laptop` by default.
```